src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Message.java

```diff
@@ -16,6 +17,11 @@
     private final HealthAuthorityNameAndAddress healthAuthorityNameAndAddress =
         HealthAuthorityNameAndAddress.fromString(extractSegment(HealthAuthorityNameAndAddress.KEY_QUALIFIER));
 
+    @Getter(lazy = true)
+    private final Optional<RequesterNameAndAddress> requesterNameAndAddress =
```

Comment:
Do we know if an edifact message **always** contains a `RequesterNameAndAddress`?

Reply:
I've looked through the first 10 messages in the example edifact file, and this exists

Reply:
By the spec, Segment Group 1 always has a `NAD` name and address, but it could be either `PO` ordering party (requester - this one) **or** `SLA` laboratory service provider